backend/server/src/db/migrations/3-realms.sql

```diff
@@ -7,16 +7,19 @@ create table realms (
     id bigint primary key default randomized_id('realm'),
     parent bigint references realms on delete restrict,
     name text not null,
-    path_segment text not null
+    path text not null,
```

Comment:
Another alternative:

Besides the performance implications I think there is another factor at play here. This makes the realm path completely independent of the parent-child relationship. We **might** actually want that at some point, but we might also not, in which case we should think about ways to make sure it doesn't happen. :thinking:

Reply:
Another thing regarding making editing and inserting more convenient: the `default` of a column is less restricted in what kind of statements it allows. Maybe we can at least construct the path automatically on insert? :thinking: That would probably require us to split `path` in two fields, though: `path_head` and `path_tail` or something like that.

Dunno, just thoughts. :thinking: